packages/forma-36-react-components/src/components/CheckboxField/README.mdx

```diff
@@ -0,0 +1,57 @@
+# CheckboxField
+
+CheckboxField is a component that gives an possiblity to easy setup proper checkbox with it's label. Component contains also an options that allows to set helper text displayed below the label. You might see checkboxes in forms, they might be also used for filtering data on the page or terms and conditions. Everywhere where user have multiple choice you might want to use this component.
+
+## Example of usage
+
+```jsx
+import {CheckboxField} from '@contentful/forma-36-react-components';
+
+<CheckboxField
+  checked={}
+  onChange={hadnleOnChange}
+  name="checkbox name"
+  id="randomId"
+  labelText="Label text"
+  helpText="Helper text displayed below"
+/>
+```
+
+Light label example
+
+```jsx
+import {CheckboxField} from '@contentful/forma-36-react-components';
+
+<CheckboxField
+  checked={}
+  onChange={hadnleOnChange}
+  name="checkbox name"
+  id="randomId"
+  labelText="Label text"
+  helpText="Helper text displayed below"
+  labelIsLight
+/>
+```
+
+## Content recommendations
+
+ - Start every option from a capital letter
+ - Do not use commas at the end of each line
+ - If the checkbox is used to agree to terms and conditions, we recommend to use first person in the sentence
```

Comment:
@restiles what do you think about those recommendations? Does it make sense from your perspective?

Reply:
@burakukula These make total sense to me. :)

Reply:
@restiles thank you!